.ahoy.yml

```diff
@@ -128,6 +128,12 @@ commands:
     usage: Run site audit
     cmd: docker-compose exec -T test drutiny profile:run ci @self "$@"
 
+  debug:
+    usage: Enable debug configuration.
+    cmd: |
+      { ahoy run "php -v|grep -q Xdebug" && echo "Debug is already enabled"; } \
```

Comment:
despite bing a weird long line, this does multiple things:
- checks that debug is not already enabled and bails early if it is. This is very useful if a developer forgot if xdebug is currently enabled or not.
- enables xdebug and restarts containers. Once restarted, check that xdebug actually enabled by examining the output of `php -v`. This is super important as there was an upstream issue where xdebug would be either always enabled or never enabled. This check makes sure that it works as required.